.gitignore

```diff
@@ -1,4 +1,5 @@
 *.iml
 *.xml
 *.class
```

Comment:
You don't need this, as all files with extension *.class are on the target directory.